lib/src/presentations/sign_up/widgets/birthday_widget.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:jobspot/src/core/constants/constants.dart';
+
+class BirthdayWidget extends StatelessWidget {
+  const BirthdayWidget({
+    super.key,
+    this.title = "Birthday",
+    required this.onChange,
+    required this.selectedDate,
+    required this.lastDate,
+  });
+
+  final String title;
+  final Function(DateTime date) onChange;
+  final DateTime selectedDate;
+  final DateTime lastDate;
+
+  @override
+  Widget build(BuildContext context) {
+    final formater = DateFormat('dd/MM/yyyy');
```

Comment:
Should be defin a class about format date time. Example : DateTimeUtils ...

Reply:
I fixed it